### DISCUSSION TITLE
taskbar moved to top

### DISCUSSION THREAD

**Discussion by jesperprintz** [valinet/ExplorerPatcher#3188] [General]
as topic.

Suddenly when i have explorer patcher installed my taskbar is on top and i cannot move it back down.

Is there some reintroduced setting i am not aware of? I did not update explorer so it does seem wierd, same second i uninstall the patcher it goes back down like default win 11.

**Comment by pyrates999** [comment]
Right click on the taskbar and click on **Lock the taskbar** so it is not ticked.  Then you can move it.

Repeat that so it can't be moved or resized again.
